README.md

```diff
@@ -253,15 +253,17 @@ The env vars are:
 | yarn start  |  Starts the express server (requires env vars)      |
 | yarn build  |  Builds the libs (all apps) (requires env vars)     |
 
-Example: Building and running a production instance of the AMO app:
+**Example:** Building and running a production instance of the AMO app:
 
 ```
 NODE_APP_INSTANCE=amo NODE_ENV=production yarn build
 NODE_APP_INSTANCE=amo NODE_ENV=production yarn start
 ```
 
-To run the app locally in production mode you'll need to create a config file
-that looks something like this. Name it exactly as `config/local-production-amo.js`.
+**Note: To serve assets on production builds locally you must also create a local production config:**
```

Comment:
It's more than just serving assets though. This sentence is pretty much a duplicate of the one below it. Maybe the old sentence should just be marked bold instead?